### PR TITLE
Animation: Default animation value don't match AMP animations

### LINKED ISSUE #5413: Animation default easing value is linear
## Bug Description

Default easing function is linear, which looks pretty bad. @samitron7 has indicated that default should not be linear for any of the animations

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by maxyinger**
## Summary
Updates animation effects to reflect the latest updates in amp-story-animation: 
https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/animation-presets.js#L131

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
NA, Behind animation feature flag: should have new default duration values per effect & updated easings

## Testing Instructions
enable animations, go to story editor, apply each animation to an object and see that each animation effect has a different default duration and has been updated.

---

Fixes #5413

### CODE REVIEW

File: assets/src/animation/effects/drop/animationProps.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+export const DropEffectInputPropTypes = {
+  duration: PropTypes.shape(AnimationInputPropTypes),
+};
+
+export default {
+  duration: {
+    label: __('Duration', 'web-stories'),
+    type: FIELD_TYPES.NUMBER,
+    unit: _x('ms', 'Time in milliseconds ', 'web-stories'),
```
Comment (BrittanyIRL) [inline assets/src/animation/effects/drop/animationProps.js]:
```suggestion
    unit: _x('ms', 'Time in milliseconds', 'web-stories'),
```

is the trailing space intentional?

File: assets/src/animation/parts/pulse/index.js [hunk 2]
```diff
@@ -41,8 +41,8 @@ const defaults = {
 
 export function AnimationPulse({
   iterations = 1,
-  scale = 0.5,
-  easing = 'cubic-bezier(0.4, 0.0, 0.2, 1)',
+  scale = 0.05,
```
Comment (carloskelly13) [inline assets/src/animation/parts/pulse/index.js:44]: Is this correct? This feels so small, it's hard to see this on small shapes.

**Review by BrittanyIRL — APPROVED** [review]
LGTM!

**Review by carloskelly13 — APPROVED** [review]
I have one question about pulse, but otherwise this looks a lot smoother.

File: assets/src/animation/parts/pulse/index.js (hunk 2 above)
Comment (maxyinger) [reply assets/src/animation/parts/pulse/index.js:44]: yea I think it feels small too, but this is the latest for the default setting on the amp-story-animation presets:
https://github.com/ampproject/amphtml/blob/7601993eff6c4399745ae0f6dd46653c7ceaa84e/extensions/amp-story/1.0/animation-presets.js#L117

**Pull request merged**